genesyscloud/responsemanagement_responseasset/genesyscloud_responsemanagement_responseasset_proxy.go

```diff
@@ -80,6 +91,48 @@ func (p *responsemanagementResponseassetProxy) deleteRespManagementRespAsset(ctx
 	return p.deleteRespManagementRespAssetAttr(ctx, p, id)
 }
 
+func getAllResponseAssetsFn(ctx context.Context, p *responsemanagementResponseassetProxy) (*[]platformclientv2.Responseasset, *platformclientv2.APIResponse, error) {
+	var allResponseAssets []platformclientv2.Responseasset
+	var response *platformclientv2.APIResponse
+	pageSize := 100
+
+	responseAssets, resp, err := p.responseManagementApi.PostResponsemanagementResponseassetsSearch(platformclientv2.Responseassetsearchrequest{
+		PageSize:   &pageSize,
+		PageNumber: platformclientv2.Int(1),
+	}, []string{})
+	response = resp
+	if err != nil {
+		return nil, resp, fmt.Errorf("Failed to get response asset search request: %v", err)
+	}
+
+	if responseAssets.Results == nil || len(*responseAssets.Results) == 0 {
+		return &allResponseAssets, resp, nil
+	}
+	allResponseAssets = append(allResponseAssets, *responseAssets.Results...)
+
+	for pageNum := 2; pageNum <= *responseAssets.PageCount; pageNum++ {
+		responseAssets, resp, err := p.responseManagementApi.PostResponsemanagementResponseassetsSearch(platformclientv2.Responseassetsearchrequest{
```

Comment:
Does this resource upload assets to Genesys Cloud?  If so how do we pull the assets back down?

Reply:
that is being handled in the resolver in the utils file through the 
`files.DownloadExportFile(fullPath, fileName, *data.ContentLocation)`